runtime/Go/antlr/atn_config.go

```diff
@@ -198,23 +218,27 @@ func (b *BaseATNConfig) String() string {
 	return fmt.Sprintf("(%v,%v%v%v%v)", b.state, b.alt, s1, s2, s3)
 }
 
+// LexerATNConfig extends BaseATNConfig for lexers.
```

Comment:
Nit: "extends" is the wrong term/concept for Go. "embeds" would technically be the correct term.